client/ayon_blender/api/pipeline.py

```diff
@@ -296,8 +297,17 @@ def set_resolution(data):
     scene.render.resolution_y = resolution_y
 
 
+def set_unit_scale_from_settings(unit_scale_settings=None):
+    if unit_scale_settings is None:
+        return
+    unit_scale_enabled = unit_scale_settings.get("enabled")
+    if unit_scale_enabled:
+        unit_scale = unit_scale_settings.get("base_file_unit_scale")
+        bpy.context.scene.unit_settings.scale_length = unit_scale
```

Comment:
If this key did not exist, we're setting `None` to the blender value which I'm _pretty sure_ would not work? :)

```suggestion
        unit_scale = unit_scale_settings["base_file_unit_scale"]
        bpy.context.scene.unit_settings.scale_length = unit_scale
```

Should we just require the key to exist?